lib/handlers/api.ts

```diff
@@ -157,26 +171,26 @@ export class ApiHandler {
             });
     }
 
-    handleLanguages(req, res) {
+    handleLanguages(req: express.Request, res: express.Response) {
         const availableLanguages = this.usedLangIds.map(val => {
             const lang = this.languages[val];
-            const newLangObj = Object.assign({}, lang);
+            const newLangObj: Language & {defaultCompiler?: string} = Object.assign({}, lang);
```

Comment:
I'll take your word on this one. Should `defaultCompiler?` maybe go on `Language`? itself?

Reply:
Ah good question.... It looks like this field is only populated when handing off to the frontend. Would it make sense to differentiate between a back-end `Language` and front-end `Language`? 
As I think through this more I think adding `defaultCompiler?` to Langauge for now should be ok, at least for now.